.github/dependabot.yml

```diff
@@ -0,0 +1,12 @@
+version: 2
+updates:
+  - package-ecosystem: 'npm'
+    directory: '/'
+    schedule:
+      interval: daily
+      time: "03:00"
+    target-branch: main
```

Comment:
Default number of PRs is 5 so would be good to set the limit to 100. Similar to https://github.com/alphagov/di-authentication-api/blob/main/.github/dependabot.yml#L8